resources/assets/js/app.js

```diff
@@ -31,7 +31,7 @@ $(function() {
     });
 
     $('[data-dismiss=timeout]').each(function() {
-        var timeout_len = 2000,
+        var timeout_len = 1000,
```

Comment:
:) Glad you remembered we talked about this! however, a shorter timeout means some folks won't be able to read it. Rather, we need to make it dismissable. It feels like that wasn't an option when I was running into it; could you re-up the timeout, and check whether the dismiss is working? Thanks!